sgkit/stats/aggregation.py

```diff
@@ -0,0 +1,73 @@
+import dask.array as da
+import numpy as np
+import xarray as xr
+from xarray import DataArray, Dataset
+
+
+def count_alleles(ds: Dataset) -> DataArray:
+    """Compute allele count from genotype calls.
+
+    Parameters
+    ----------
+    ds : Dataset
+        Genotype call dataset such as from
+        `sgkit.create_genotype_call_dataset`.
+
+    Returns
+    -------
+    variant/allele_count : DataArray
+        Allele counts with shape (variants, alleles) and values
+        corresponding to the number of non-missing occurrences
+        of each allele.
+
+    Examples
+    --------
+
+    >>> import sgkit as sg
+    >>> from sgkit.testing import simulate_genotype_call_dataset
+    >>> ds = simulate_genotype_call_dataset(n_variant=3, n_sample=2, seed=1)
+    >>> ds['call/genotype'].to_series().unstack().astype(str).apply('/'.join, axis=1).unstack()
+    samples 0   1
+    variants
+    0       1/0	1/0
+    1       1/0	1/1
+    2       0/1	1/0
+    3       0/0	0/0
+
+    >>> sg.allele_count(ds)
```

Comment:
This should be `sg.count_alleles(ds)` now. When I run it I don't see the array (since it's a Dask array, not a NumPy array), just 
```
<xarray.DataArray 'variant/allele_count' (variants: 3, alleles: 2)>
dask.array<getitem, shape=(3, 2), dtype=int64, chunksize=(3, 2), chunktype=numpy.ndarray>
Dimensions without coordinates: variants, alleles
```

Also, the output from the unstack should have 3 variants, not 4.

Reply:
Whoops, ty!  Will change that soon.

Reply:
https://github.com/pystatgen/sgkit/pull/36/commits/52f9d5598e2ea66f834733ca3fee21ea14ec89da

Reply:
I don't want to hold this up, but it would be easy to enable doctest testing by adding the following to `setup.cfg`.

```
[tool:pytest]
addopts = --doctest-modules
```

We can do that in another PR though if you like.

Reply:
I'll give it a shot and see what happens

Reply:
I had to add `# doctest: +NORMALIZE_WHITESPACE` to get the outputs to match but it worked!  I'm a little hesitant to say that's a good global setting since I know they only use it when necessary in the dask codebase so presumably that option is better left to individual lines like this.

Changes in: https://github.com/pystatgen/sgkit/pull/36/commits/ea0185ec4684a3591f554522186166474e9218db

Reply:
+1 this looks great